website/static/js/osfHelpers.js

```diff
@@ -701,7 +701,22 @@ var confirmDangerousAction = function (options) {
 
     bootbox.dialog(bootboxOptions);
 };
-
+/**
+ * Maps an object to an array of {key: KEY, value: VALUE} pairs
+ *
+ * @param {Object} obj
+ * @returns {Array} array of key, value pairs
+ **/
+var iterObject = function(obj) {
```

Comment:
Yes, this helper function was present in the branch at some point in the past but seems to have been removed without checking where it was used. This just adds it back in